chrome/nav.js

```diff
@@ -148,8 +148,10 @@ function applyConfiguration() {
 		updatePathfinding();
 
 		let ukey = Universe.getServer( doc ).substr( 0, 1 );
-		let name = ukey + 'path';
+		let name = ukey + 'savedPath';
 		chrome.storage.local.get( name , updateRoutePlanner );
+		let missionSet = [ ukey + 'm' + mission.locId, ukey + 'mlist' ];
```

Comment:
mission is not necessarily defined. not sure if this breaks functionality, it does leave an error though.

Reply:
I see. It should be as the other one. Will fix :)